walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -37,31 +37,76 @@
          return eventsAsTimeRange;
       }
     }
+    // convert events to list representation
+    List<Event> eventsAsList = new ArrayList<Event>();
+    for (Event event: events) {
+      eventsAsList.add(event);
+    }
+    
+    // if there are no mandatory attendees, make optional attendees mandatory attendees
+    if (request.getAttendees().isEmpty()) {
+      List<String> optionalAttendees = new ArrayList<String>();
+      for(String optionalAttendee: request.getOptionalAttendees()) {
```

Comment:
nit: fix spacing here 
`for (String optionalAttendee : request.getOptionalAttendees()) {`